Buildings/Experimental/OpenBuildingControl/CDL/Logical/Validation/TrueFalseHold.mo

```diff
@@ -0,0 +1,126 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.Logical.Validation;
+model TrueFalseHold "Validation model for the TrueFalseHold block"
+extends Modelica.Icons.Example;
+
+  Sources.BooleanPulse booPul(
```

Comment:
This would be good for the more complex tests (as is done in Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.Validation.VAVSingleZoneTSupSet_T), but I rather focus the time on finishing the sequences and demonstrating closed loop response, which is more time critical.